lib/metrics.rb

```diff
@@ -1,6 +1,6 @@
 class Metrics
   def self.with_object(object)
-    prefix = object.class.name.underscore.tr("/", ".")
+    prefix = object.class.name.underscore.tr("/", "_")
```

Comment:
Wondering if we should get rid of `with_object` and `with_prefix` and turn this into a Singleton class and have the `event` name provided only at the method call. The prefix is helpful in some parts of the codebase because you can use the class name as a prefix.